src/Illuminate/Foundation/Console/MailMakeCommand.php

```diff
@@ -68,6 +73,30 @@ protected function writeMarkdownTemplate()
         $this->files->put($path, file_get_contents(__DIR__.'/stubs/markdown.stub'));
     }
 
+    /**
+     * Write the Blade template for the mailable.
+     *
+     * @return void
+     */
+    protected function writeView()
+    {
+        $path = $this->viewPath(
+            str_replace('.', '/', $this->getView()) . '.blade.php'
+        );
+
+        if (!$this->files->isDirectory(dirname($path))) {
+            $this->files->makeDirectory(dirname($path), 0755, true);
+        }
+
+        $stub = str_replace(
+            '{{ quote }}',
+            Inspiring::quotes()->random(),
+            file_get_contents(__DIR__ . '/stubs/view.stub')
```

Comment:
The generated blade view stub here has a `<div>` in it. Wouldn't that be a little strange for emails? We might need a new mailable view stub or something to suffice.